src/main/java/William.java

```diff
@@ -0,0 +1,50 @@
+import java.io.FileNotFoundException;
+import java.util.List;
+
+import exceptions.WilliamException;
+import tasks.Task;
+import utilities.Parser;
+import utilities.Storage;
+import utilities.TaskList;
+import utilities.Ui;
+
+/**
+ * Main class for this chatbot (the other classes have been divided into
+ * packages: commands, exceptions, tasks and utilities)
+ */
+public class William {
+
```

Comment:
Basic indentation is all 4 spaces, which follows the Java coding standard, Nice!